packages/core/src/Stores/traders-hub-store.js

```diff
@@ -546,7 +556,21 @@ export default class TradersHubStore extends BaseStore {
             account => account.platform === CFD_PLATFORMS.CTRADER
         );
     }
-
+    /**
+     * Get default Jurisdiction for MT5 product types
+     * Product types = Standard /Financial /Swap Free /Zero Spread/
+     *
+     */
+    getDefaultJurisdiction() {
+        const { trading_platform_available_accounts } = this.root_store.client;
+        const { product } = this.root_store.modules.cfd;
+
+        const default_jurisdiction = trading_platform_available_accounts.filter(
+            available_account =>
+                available_account.product === product && available_account.is_default_jurisdiction === 'true'
```

Comment:
is is expected to check the equality with 'true' string?

Reply:
yes. API response is string